src/dos/dos_devices.cpp

```diff
@@ -375,6 +375,61 @@ DOS_File &DOS_File::operator=(const DOS_File &orig)
 	return *this;
 }
 
+void DOS_SetMediaDelay(const bool use_media_delay)
+{
+	auto get_delay_percent = []() {
+		LOG_MSG("machine = %d", machine);
+		switch (machine) {
+		case MCH_HERC: return 300;
+		case MCH_CGA:
+		case MCH_TANDY:
+		case MCH_PCJR: return 150;
+		case MCH_EGA: return 125;
+		case MCH_VGA:
+		default: return 100;
+		}
+	};
+
+	DOS_File::MediaDelayGuard::SetDelayPercent(
+	        use_media_delay ? get_delay_percent() : 0);
+
+	LOG_MSG("DOS: Media will %s delays",
+	        use_media_delay ? "use realistic" : "run without");
```

Comment:
We don't really use future tense in the logs; we state things:

<img width="858" alt="image" src="https://github.com/user-attachments/assets/67e07823-b15f-490b-90cf-d529d10a5654">

Can you reword please to "Using realistic disk access delays" and "Realistic disk access delays disabled" or something?